### ISSUE TITLE
Show page number / navigation in the toolbar

### ISSUE THREAD

**Issue by ericschdt** [xournalpp/xournalpp#2440]
**Is your feature request related to a problem? Please describe.**
I use Xournal++ for annotating a PDF with around 200 pages. It'd be convenient to see the current page number and also to quickly jump to another page number by typing it in an input field.

**Describe the solution you'd like**
There could be a toolbar item for it. That how it looks like for Okular:
![Screenshot_20201124_175903](https://user-images.githubusercontent.com/43318127/100126854-632f6080-2e76-11eb-8b48-9fe72e5b23cd.png)

**Comment by rolandlo** [comment]
That's already there. For example in the Xournal++ toolbar you find it on the bottom left:
![grafik](https://user-images.githubusercontent.com/40485433/100137125-4b62d700-2e8c-11eb-869f-ba26df3b7304.png)

**Comment by ericschdt** [comment]
The problem is, it's in the `Xournal++` toolbar, but it does not show up in the `toolbar items overview` dialog.
Thus I switched to `Xournal++` toolbar, dragged and dropped into the `toolbar items overview` dialog and switched back to the `right hand note taking` toolbar.
Finally I tried to drag and drop it into the top toolbar. This however, didn't work: It's not shown and in the `toolbar items overview` dialog it's gone! :-(

Maybe I should open an issue about the toolbar configuration UX first.
